lib/bridge.js

```diff
@@ -702,10 +702,10 @@ Bridge.prototype._onAliasQuery = function(alias) {
             if (shouldAdvertiseGroup && self.groupId) {
                 return self._createGroupIfNotExists().then(function() {
                     return self._botClient.getStateEvent(roomId, "m.room.related_groups", "");
-                }).then(function(currentState) {
+                }).then(function(relatedGroupsContent) {
                     var groups = [];
-                    if (currentState && currentState.getContent()) {
-                        groups = currentState.getContent().groups || [];
+                    if (relatedGroupsContent && relatedGroupsContent.groups) {
```

Comment:
I think we also should validate that `groups` is actually an array, for the sake of being paranoid.